src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/FindNamespacesCommandResolver.java

```diff
@@ -13,12 +13,9 @@
 @ApplicationScoped
 public class FindNamespacesCommandResolver implements CommandResolver<FindNamespacesCommand> {
 
-  private final SchemaManager schemaManager;
+  @Inject CQLSessionCache cqlSessionCache;
```

Comment:
We can leave this as-is, or pass through constructor, I don't care either way :)
(this way is bit more compact)